mock-caas/src/main/java/com/sequenceiq/caas/grpc/service/MockUserManagementService.java

```diff
@@ -491,25 +499,52 @@ public void getActorWorkloadCredentials(com.cloudera.thunderhead.service.userman
         final int keyType17 = 17;
         final int keyType18 = 18;
         final int saltType = 4;
-        GetActorWorkloadCredentialsResponse.Builder respBuilder = GetActorWorkloadCredentialsResponse.getDefaultInstance().toBuilder();
-        respBuilder.addKerberosKeysBuilder(0)
-                .setSaltType(saltType)
-                .setKeyType(keyType17)
-                .setKeyValue("testKeyValue17")
-                .setSaltValue("NonIodizedGrainOfSalt")
-                .build();
-
-        respBuilder.addKerberosKeysBuilder(1)
-                .setSaltType(saltType)
-                .setKeyType(keyType18)
-                .setKeyValue("testKeyValue18")
-                .setSaltValue("IodizedGrainOfSalt")
-                .build();
+        final int hashsaltlength = 32;
+        final int keysaltlength = 8;
+        final String password = "Password123!";
```

Comment:
it looks like the response is the same regardless of the request. can we build it once rather than for each request?